style.css

```diff
@@ -0,0 +1,318 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  font-family: 'Lato', sans-serif;
+}
+
+.no-scroll {
+  overflow: hidden;
+}
+
+.top-header {
+  display: none;
+}
+
+.logo {
+  display: none;
+}
+
+.desktop-nav {
+  display: none;
+}
+
+.menu-icon {
+  background: transparent;
+  padding-left: 10px;
+}
+
+.menu-icon img {
+  width: 3em;
+}
+
+.hide {
+  display: none;
+}
+
+.visible {
+  visibility: hidden;
+}
+
+.mobile-menu {
+  width: 100%;
+  position: absolute;
+  background-color: black;
+  opacity: 0.8;
+  height: 100vh;
+  top: 0;
+  right: 0;
+  text-align: center;
+}
+
+.mobile-menu ul {
+  margin-top: 150px;
+}
+
+.mobile-menu li {
+  margin-bottom: 20px;
+}
+
+.mobile-menu li a {
+  color: #fff;
+  position: relative;
+  cursor: pointer;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 25px;
+  padding: 10px;
+  transition: transform 0.5s;
+}
+
+.mobile-menu li a:hover {
+  color: #ec5242;
+}
+
+.cancle-mobile {
+  position: absolute;
+  top: 25px;
+  right: 20px;
+}
+
+#hero {
+  background-image: url(./images/bg.jpg);
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.hero-container {
+  width: 80%;
+  margin-top: 20px;
+  padding-left: 20px;
+}
+
+.hero-container h2 {
+  margin-top: 100px;
+  font-weight: 400;
+  font-size: 50px;
+  color: #ec5242;
+}
+
+.details {
+  width: 98%;
+  font-weight: 900;
+  font-size: 35px;
+  margin-top: 10px;
+  color: transparent;
+  -webkit-text-stroke: 1px #ec5242;
+  background-image: url(./images/text-bg.jpg);
+  -webkit-background-clip: text;
+  background-position: 0 0;
+  animation: back 20s linear infinite;
+}
+
+.info {
+  width: 85%;
+  margin-top: 0.5em;
+  font-weight: 400;
+  font-size: 16px;
+  background-color: darkgray;
+  border: 5px solid white;
+  padding: 5px;
+}
+
+.hero-container h3 {
+  font-weight: 800;
+  margin-top: 15px;
+  font-size: 20px;
+  color: rgb(151, 129, 129);
+}
+
+.hero-container h4 {
+  font-weight: 500;
+  font-size: 12px;
+  width: 100%;
+  color: rgb(151, 129, 129);
+
+  /* padding-bottom: 40px; */
+  margin-bottom: 100px;
+
+  /* margin: 0 15px 0 15px; */
+}
```

Comment:
- [ OPTIONAL] Please remove all unused code left as comments in your codebase because hey make your project look messy. If you need to review a previous version of your code, you can always use git history. 👍🏾